README.md

```diff
@@ -60,7 +63,10 @@ To Install:
 
 #### **What is the docker container?**
 
-The docker container is a self-contained environment in which you can run openworm simulations.  It's fully setup to get you started by following the steps above.  At the moment, it runs simulations and produces visualizations for you, but these visualizations must be viewed outside of the docker container.  
+The docker container is a self-contained environment in which you can run OpenWorm simulations.  It's fully setup to get you started by following the steps above.  At the moment, it runs simulations and produces visualizations for you, but these visualizations must be viewed outside of the docker container.  While you do not need to know much about Docker to use OpenWorm,
```

Comment:
not your change, but should be, "It's fully set[ ]up" not "It's fully setup".

The tut looks OK, but could definitely stop at section 2.5. Running on AWS isn't something you need to get going.